Road/source/data/get_data.py

```diff
@@ -0,0 +1,21 @@
+from telethon import TelegramClient,events
+import csv
+
+api_id = 28394694 
+api_hash = '4c0b02c61f2626bd3f0d1b58883ec759'
+client = TelegramClient('session_name', api_id, api_hash) # create a client object to connect to telegram using my account info
```

Comment:
could you pls add the comments before the code line, not in front of it? to make sure we're all following the same coding style.

Reply:
in  the convention we follow the inline comments should be in the same line of code  (pep8 convention)